calliope/example_models/national_scale/scenarios.yaml

```diff
@@ -2,8 +2,8 @@
 # Scenarios are optional, named combinations of overrides
```

Comment:
Do we need these scenarios any more, given that we have our minimal examples?

Reply:
I would leave them until we actually have consistent minimal examples -- right now, the minimal examples are only visible if you actually dig through the source code looking at the tests (and who does _that_?)